changelogs/unreleased/fix-tuple-leak-in-lua-key_def.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/lua
+
+* Fixed a leak of a tuple object in `key_def:compare_with_key(tuple, key)`,
```

Comment:
You might notice that we use imperative in all changelog records. Would be good to be consistent.

Reply:
In the past release notes:

- Imperative mood is used when we start to do X: say, 'Deploy packages for Fedora 32 (gh-4966)'.
- Past tense is used to describe a past (changed / fixed) behaviour: say, 'Fixed a false positive panic when yielding in debug hook (gh-5649)'.
- Present tense is used to describe a new behaviour: say, 'Data changes in read-only mode are now forbidden (gh-5231)'.

Unlike using an imperative mood, this approach results in a text that is 'written for a human', like an article about changes. For me, it looks good.

However, I'm not an expert neither in grammar, nor in documentation writting, so it is better to ask @tarantool/doc team.

> You might notice that we use imperative in all changelog records. Would be good to be consistent.

There is no consistency in current unreleased changelog entries, but I hope we'll proof-read them before a next release and adjust to a certain style. We can elaborate some preliminary agreements with the documentation team here.

Reply:
@NickVolynkin said that he will participate in the discussion after a week.